panoptes_client/panoptes.py

```diff
@@ -101,8 +103,10 @@ def __init__(
             'PANOPTES_ENDPOINT',
             'https://www.zooniverse.org'
         )
-        self.username = username or os.environ.get('PANOPTES_USERNAME')
-        self.password = password or os.environ.get('PANOPTES_PASSWORD')
+        self.username = None
+        self.password = None
+        self._auth(username, password, login)
```

Comment:
Looks like the arguments are in the wrong order here compared with the definition of `_auth`.